applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -55,9 +92,9 @@ def __init__(self, model, custom_settings):
         self._is_printing_rank = True
 
         # Set the element and condition names for the replace settings
-        # TODO: Create a formulation class helper as soon as there is more than one element is present
-        self.element_name = "IncompressiblePotentialFlowElement"
-        self.condition_name = "PotentialWallCondition"
+        self.embedded_formulation = PotentialFlowFormulation(self.settings["formulation"])
```

Comment:
Is there a reason why you called this variable embedded_formulation? I would call it just formulation since it can be embedded or not embedded.

Reply:
Probably the cause is that I called it  `self.embedded_formulation` in the embedded solver. I agree with @inigolcanalejo about calling it `self.formulation`.

Actually, I will change it in the embedded solver just in case there is an eventual derivation from it (even though the proper way to retrieve the formulation name would be to implement a `GetFormulationName()` method).

Reply:
Yes, my bad! This was due to copy&pasting the class from the fluid and I forgot to change the name!